documentation/Installing-Clear-Containers-on-RHEL.md

```diff
@@ -1,206 +1,58 @@
-$ curl -L -O http://www.mirrorservice.org/sites/ftp.gnu.org/gnu/gcc/gcc-6.2.0/gcc-6.2.0.tar.gz
-$ tar xvf gcc-6.2.0.tar.gz
-$ pushd gcc-6.2.0
-$ ./configure --enable-languages=c --disable-multilib --prefix=/usr/local/gcc-6.2.0
-$ make -j5 && sudo make install
-$ popd
-
-```
-
-## Install qemu-lite
-
-Although this package installs the `qemu-lite` version of `qemu`, the binary will still be called `qemu-system-x86_64`.
-Requires at least `gcc` 5.
+For more information on Docker installation, see:
+- https://docs.docker.com/engine/installation/linux/rhel/
 
-```
-$ export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH
-$ export PATH=/usr/local/gcc-6.2.0/bin:$PATH
-$ sudo yum install libcap-ng-devel pixman-devel libattr-devel libcap-devel
-$ curl -L -O https://github.com/01org/qemu-lite/archive/741f430a960b5b67745670e8270db91aeb083c5f.tar.gz
-$ tar xvf 741f430a960b5b67745670e8270db91aeb083c5f.tar.gz
-$ pushd qemu-lite-741f430a960b5b67745670e8270db91aeb083c5f
-$ CC=/usr/local/gcc-6.2.0/bin/gcc ./configure \
-	--disable-bluez         --disable-brlapi \
-	--disable-bzip2         --disable-curl \
-	--disable-curses        --disable-debug-tcg \
-	--disable-fdt           --disable-glusterfs \
-	--disable-gtk           --disable-libiscsi \
-	--disable-libnfs        --disable-libssh2 \
-	--disable-libusb        --disable-linux-aio \
-	--disable-lzo           --disable-opengl \
-	--disable-rbd           --disable-qom-cast-debug \
-	--disable-rdma          --disable-sdl \
-	--disable-seccomp       --disable-slirp \
-	--disable-snappy        --disable-spice \
-	--disable-strip         --disable-tcg-interpreter \
-	--disable-tcmalloc      --disable-tools \
-	--disable-tpm           --disable-usb-redir \
-	--disable-uuid          --disable-vnc \
-	--disable-vte           --disable-vnc-{jpeg,png,sasl} \
-	--disable-xen           --enable-attr \
-	--enable-cap-ng         --enable-kvm \
-	--enable-virtfs         --target-list=x86_64-softmmu \
-	--extra-cflags="-fno-semantic-interposition -O3 -falign-functions=32" \
-	--datadir=/usr/local/share/qemu-lite \
-	--libdir=/usr/local/lib64/qemu-lite \
-	--libexecdir=/usr/local/libexec/qemu-lite
-$ make -j5 && sudo make install
-$ popd
-```
+## Installation steps
 
-## Install container kernel and rootfs
+The installation script can be found in the installation subdirectory of the cc-oci-runtime source tree. In order to install Clear Containers on RHEL 7.3 run the following command:
 
 ```
-$ sudo yum-config-manager --add-repo http://download.opensuse.org/repositories/home:/clearlinux:/preview:/clear-containers-2.1/RHEL_7/home:clearlinux:preview:clear-containers-2.1.repo
-$ sudo yum update
-$ sudo yum install linux-container clear-containers-image clear-containers-selinux
+$ ./installation/rhel-setup.sh
 
 ```
+Note:
+This script will install all the packages required by cc-oci-runtime. This script uses sudo, and will therefore potencially ask you for your password to proceed.
```

Comment:
Change to: This script uses sudo and could potentially ask for the password to proceed.